File: android/employer/app/src/main/java/com/workforcex/employer/api/Verification.java
package com.workforcex.employer.api;

import com.google.gson.annotations.SerializedName;

public class Verification {
    @SerializedName("id")
    public String id;

    @SerializedName("user")
    public User user;

    @SerializedName("verificationType")
    public String verificationType;

    @SerializedName("status")
    public String status;

    public static class User {
        @SerializedName("mobileNumber")
        public String mobileNumber;
    }
}

File: android/employer/shared_employer/src/commonMain/kotlin/com/workforcex/shared_employer/models/Verification.kt
package com.workforcex.shared_employer.models

import com.google.gson.annotations.SerializedName

data class Verification(
    @SerializedName("id")
    val id: String,

    @SerializedName("user")
    val user: User,

    @SerializedName("verificationType")
    val verificationType: String,

    @SerializedName("status")
    val status: String,

    @SerializedName("documents")
    val documents: List<Document> = emptyList()
) {
    data class User(
        @SerializedName("id")
        val id: String,
        @SerializedName("mobileNumber")
        val mobileNumber: String,
        @SerializedName("name")
        val name: String?
    )
}